core/conversion/evaluators/aten.cpp

```diff
@@ -465,7 +465,21 @@ auto aten_registrations TRTORCH_UNUSED =
                       LOG_WARNING("Warning from TorchScript: " << *warning);
                       return {};
                     },
-                    EvalOptions()});
+                    EvalOptions()})
+        .evaluator({c10::Symbol::fromQualString("aten::arange"),
+                    [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
+                      int end_scalar = 0;
+                      if (args.at(n->input(0)).IValue()->isInt()) {
```

Comment:
what about the cases where torch arange takes in `start` and `step` values? Is that handled or am I missing something here ?

Reply:
ok, I will implement that part too